Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Publish&Install_configuration.yml

```diff
@@ -0,0 +1,780 @@
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsString
+          left:
+            value:
+              complex:
+                root: CheckPoint
+                accessor: list-gateways.type
+            iscontext: true
+          right:
+            value:
+              simple: gateway
+    view: |-
+      {
+        "position": {
+          "x": -200,
+          "y": 1400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "6":
+    id: "6"
+    taskid: a958142c-d8ed-4d3f-8e26-3a04adb263e8
+    type: regular
+    task:
+      id: a958142c-d8ed-4d3f-8e26-3a04adb263e8
+      version: -1
+      name: checkpoint-install-policy
+      description: Install Policy  Changes to Checkpoint gateways. Default is "Standard".
+      script: '|||checkpoint-install-policy'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      access: {}
+      policy_package:
+        simple: Standard
+      session_id:
+        simple: ${inputs.sid}
+      targets:
+        complex:
+          root: CheckPoint
+          accessor: gateways.type
+    continueonerror: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -680,
+          "y": 1740
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "7":
+    id: "7"
+    taskid: d82f1865-8c6b-4b1d-825f-7b61fed6fdd9
+    type: regular
+    task:
+      id: d82f1865-8c6b-4b1d-825f-7b61fed6fdd9
+      version: -1
+      name: checkpoint-show-packages
+      description: Can be used to pass/fetch  relevant packages to install.
+      script: CheckPoint_FW|||checkpoint-packages-list
+      type: regular
+      iscommand: true
+      brand: CheckPoint_FW
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      limit: {}
+      offset: {}
+      session_id:
+        simple: ${inputs.sid}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -680,
+          "y": 1580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "8":
+    id: "8"
+    taskid: 9b711228-0af3-42a7-8498-d913e684ea50
+    type: regular
+    task:
+      id: 9b711228-0af3-42a7-8498-d913e684ea50
+      version: -1
+      name: checkpoint-show-gateways-and-servers
+      description: Show relevant Servers / Instances to Install policy.
+      script: CheckPoint_FW|||checkpoint-gateways-list
+      type: regular
+      iscommand: true
+      brand: CheckPoint_FW
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      limit: {}
+      offset: {}
+      session_id:
+        simple: ${inputs.sid}
+    continueonerror: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -200,
+          "y": 1225
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "9":
```

Comment:
If the integration is not enabled do you still need to logout?

Reply:
fixed